### DISCUSSION TITLE
UE向けNotionドキュメントのリンク切れ

### DISCUSSION THREAD

**Discussion by abricheese** [pixiv/vroid-sdk-developers#141] [General]
下記のUnreal Engine向けのドキュメントにて、5か所ほどリンク切れして「アクセス不可」となっていましたので報告しておきます。
https://vroid.notion.site/VRoid-SDK-for-Unreal-Engine-21321b5e31b247e99600f270b3414d4e

![image](https://github.com/pixiv/vroid-sdk-developers/assets/136788982/21444682-11b6-4006-9413-97e6d370ad59)

ご確認のほどよろしくお願いいたします。

**Comment by ytomi-pixiv** [comment]
ご指摘ありがとうございます。

報告いただき大変助かりました。
先ほど、修正いたしました。ご確認いただけますと幸いです。
